autotest.pm

```diff
@@ -45,7 +45,7 @@ sub loadtest {
     my $category = $1;
     my $name     = $2;
     my $test;
-    my $fullname = "$category-$name";
+    my $fullname = "$category-$name-$arg";
```

Comment:
@coolo this should take care to name the module unambiguously and correspondingly for the webui.